app/src/main/java/org/simple/clinic/drugs/PrescribedDrug.kt

```diff
@@ -140,5 +140,8 @@ data class PrescribedDrug(
       WHERE isDeleted = 1 AND syncStatus == 'DONE'
     """)
     fun purgeDeleted()
+
+    @Query("UPDATE PrescribedDrug SET durationInDays = :durationInDays, updatedAt = :updatedAt WHERE uuid = :id")
+    fun updateDrugDuration(id: UUID, durationInDays: Int, updatedAt: Instant)
```

Comment:
We'll also need to set the `syncStatus` of the record to `PENDING` since there's no guarantee that sync will not run before completing the teleconsult prescription saving.

Reply:
OMG, yes. I completely forgot about that.